src/shared_modules/indexer_connector/src/indexerConnector.cpp

```diff
@@ -261,14 +274,21 @@ void IndexerConnector::diff(const nlohmann::json& responseJson,
 
     if (!bulkData.empty())
     {
+        const auto onSuccess = [](const std::string& response)
+        {
+            logDebug2(IC_NAME, "Response: %s", response.c_str());
+        };
```

Comment:
unused

```suggestion

```

Reply:
Instead I added the lambda to improve debugging